tests/conftest.py

```diff
@@ -20,36 +20,39 @@ def capture_logs(caplog):
 
 
 @pytest.fixture
-def example_model(tmp_path):
+def tmp_model(tmp_path):
+   """Example resqpy model in a temporary directory unique to each test"""
+
+   model = Model(create_basics=True)
+   model_path = str(tmp_path / 'tmp_model.epc')
+   model.set_epc_file_and_directory(model_path)
+   model.create_hdf5_ext(add_as_part=True, file_name=model_path[:-4] + '.h5')
```

Comment:
there is now a convenience function in model.py: new_model(epc_path) which does lines 26, 28 & 29